node/heartbeat/monitor.go

```diff
@@ -120,7 +123,7 @@ func (m *Monitor) ProcessReceivedMessage(message p2p.MessageP2P) error {
 			m.heartbeatMessages[string(hb.Pubkey)] = pe
```

Comment:
lines 110-114 are now useless. Also peerMessenger is useless and func PeerAddress from interface.go

Reply:
removed